src/videojs-contrib-hls.js

```diff
@@ -325,6 +338,24 @@ class HlsHandler extends Component {
     this.on(this.tech_, 'play', this.play);
   }
 
+  setOptions_() {
+    // defaults
+    this.options_.withCredentials = this.options_.withCredentials || false;
+
+    // start playlist selection at a reasonable bandwidth for
+    // broadband internet
+    // 0.5 Mbps
```

Comment:
This has always bothered me.. It's 0.5**MB/s** not **mbps** - I know you didn't write this comment but I would be very appreciative if you could fix it!